rshim.spec.in

```diff
@@ -57,6 +57,8 @@ make
 %{__install} -m 0644 etc/rshim.conf %{buildroot}%{_sysconfdir}
 %{__install} -m 0755 scripts/bfb-install %{buildroot}%{_sbindir}
 %{__install} -m 0755 scripts/bf-reg %{buildroot}%{_sbindir}
+mkdir -p %{buildroot}%{_datadir}/selinux/packages
```

Comment:
Can we use "%{__install} -d" here to be consistent?

Reply:
Updated with new forced push.